### PR TITLE
Remove replace attribute from all directives

### PR DESCRIPTION
**Pull request by jmuharsky**
The .replace property was [deprecated](https://github.com/angular/angular.js/commit/eec6394a342fb92fba5270eee11c83f1d895e9fb) due to problems with merging attributes (which are blocking adoption of Angular Material 1.0).  It has been removed from all directives, and in the cases of layout elements, the wrapper <div> elements were removed.

### CODE REVIEW

**Comment by jmuharsky** [comment]
Unassigning due to lingering layout bug.

**Comment by jmuharsky** [comment]
This PR, along with PR #254 update PKX to use Angular Material 1.0.5.  The result of these branches can be seen on [Testing Release v26.7](https://26-7-dot-perfkit-explorer.appspot.com/explore?dashboard=5714163003293696).

File: client/components/widget/widget-directive.css [hunk 1]
```diff
@@ -21,12 +21,12 @@
...
 }
 
-.pk-widget:not(.pk-widget-selected) .pk-widget-body:hover {
+.pk-widget > div:not(.pk-widget-selected) .pk-widget-body:hover {
   border: 1px dashed #ccc;
   margin: 15px;
 }
 
-.pk-widget:not(.pk-widget-selected) .pk-widget-body:not(:hover) {
+.pk-widget> div:not(.pk-widget-selected) .pk-widget-body:not(:hover) {
```
Comment (klausw) [inline client/components/widget/widget-directive.css]: inconsistent whitespace around '>'.

**Comment by klausw** [comment]
LGTM

**Pull request merged**